modules/my_health/docs/openapi.yaml

```diff
@@ -337,22 +525,96 @@ paths:
               schema:
                 $ref: ./schemas/Errors.yml
           description: Forbidden
-      summary: /v1/medical_records/labs_and_tests/diagrep/{id}
-  /v1/medical_records/labs_and_tests/docref/{id}:
+      summary: /v1/medical_records/labs_and_tests/{id}
+  /v1/medical_records/military_service:
     get:
-      description: |
-        Get a patient labs and tests record of type DocumentReference. This is for EKG and Radiolody results.
-      operationId: get_labs_and_tests_docref
+      description: Get the current user's military service
+      operationId: get_military_service
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: string
+                description: A long string containing the user's military service report
+          description: OK
+        "401":
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Unauthorized
+        "403":
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Forbidden
+      summary: /v1/medical_records/military_service
+  /v1/medical_records/patient:
+    get:
+      description: Get the current user's patient account information, including a list of treatment facilities.
+      operationId: get_patient
       responses:
         "200":
           content:
             application/json:
               schema:
                 type: object
                 description: >
-                  FHIR DocumentReference. The schema can be found at
-                  <a href='https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition-VA.MHV.PHR.ecg.html'>EKG</a>,
-                  <a href='https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition-VA.MHV.PHR.imaging.html'>Radiology</a>
+                  The returned object is a 
+                  <a target="_blank" href='https://github.com/department-of-veterans-affairs/mhv-np-common-api/blob/development/src/main/java/gov/va/med/mhv/common/api/dto/PatientDTO.java'>PatientDTO object</a>.
```

Comment:
Is this right? Same with the one on line 593.

Reply:
It is linking to the right reference. The Git diff somehow got very mismatched here so it looks like I'm changing it from a FHIR reference to a Java reference but the two Patient endpoint entries are brand new.

I am linking out to the MHV objects here because (1) this API is currently functioning as a straight pass-through, and (2) this API currently returns a very complex nested object.

In the future we plan to transform this object inside vets-api into something much smaller, and will then include a schema within the Swagger doc itself.